recipes/cuda-nvcc-impl/meta.yaml

```diff
@@ -0,0 +1,188 @@
+        - test -f $PREFIX/nvvm/bin/cicc                           # [linux]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: Architecture independent part of CUDA NVCC compiler.
+      description: |
+        Compiler for CUDA applications.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-nvcc-dev_{{ target_platform }}
+    build:
+      noarch: generic
+    run_exports:
+      strong:
+        - cuda-version >={{ cuda_version }},<{{ cuda_version.split(".")[0]|int + 1 }}
+    files:
+      # targets/{{ target_name }}/bin/nvcc is only a symlink
+      - targets/{{ target_name }}/bin      # [linux]
+      - targets/{{ target_name }}/nvvm     # [linux]
+      - targets/{{ target_name }}/include  # [linux]
+      - targets/{{ target_name }}/lib      # [linux]
+      - Library\bin      # [win]
+      - Library\nvvm     # [win]
+      - Library\include  # [win]
+      - Library\lib      # [win]
+    requirements:
+      build:
+      host:
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - cuda-version {{ cuda_version }}
+        - cuda-nvcc-tools {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+      run_constrained:
+        - gcc_impl_{{ target_platform }} >=6,<13  # [linux]
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
+    test:
+      commands:
+        - test -L $PREFIX/targets/{{ target_name }}/bin/nvcc      # [linux]
+        - test -L $PREFIX/targets/{{ target_name }}/nvvm          # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/lib/libnvptxcompiler_static.a     # [linux]
+        - test -f $PREFIX/targets/{{ target_name }}/include/nvPTXCompiler.h           # [linux]
+        - if not exist %LIBRARY_LIB%\nvptxcompiler_static.lib exit 1  # [win]
+        - if not exist %LIBRARY_INC%\nvPTXCompiler.h exit 1           # [win]
+    about:
+      home: https://developer.nvidia.com/cuda-toolkit
+      license_file: LICENSE
+      license: LicenseRef-NVIDIA-End-User-License-Agreement
+      license_url: https://docs.nvidia.com/cuda/eula/index.html
+      summary: Target architecture dependent parts of CUDA NVCC compiler.
+      description: |
+        Compiler for CUDA applications.
+      doc_url: https://docs.nvidia.com/cuda/index.html
+
+  - name: cuda-nvcc-impl
+    build:
+      # libnvvm.so gets corrupted by patchelf. No need to relocate as it is already relocatable
+      binary_relocation: False
+    run_exports:
+      strong:
+        - cuda-version >={{ cuda_version }},<{{ cuda_version.split(".")[0]|int + 1 }}
+    files:
+      - include/crt                    # [linux]
+      - include/fatbinary_section.h    # [linux]
+      - include/nvPTXCompiler.h        # [linux]
```

Comment:
Are these three symlinks needed?

Reply:
They shouldn't be needed based on testing that happened against prior iterations without them. 
Removed them here now.